opentreemap/opentreemap/settings/default_settings.py

```diff
@@ -406,7 +406,7 @@
 
 # We have... issues on IE9 right now
 # Disable it on production, but enable it in Debug mode
-IE_VERSION_MINIMUM = 9 if DEBUG else 10
+IE_VERSION_MINIMUM = 9 if DEBUG else 11
```

Comment:
I tested this with BrowserStack and saw the redirect working in IE 10.

We have a hard-coded version "10" in our `unsupported.html` template that we should change as well.

https://github.com/OpenTreeMap/otm-core/blob/77f662725fc4bd3ff15e8a6817f9799757db514f/opentreemap/treemap/templates/treemap/unsupported.html